tests/DB.test.js

```diff
@@ -2,21 +2,21 @@ const db = require('../database/list.js');
 
 test('the data is an array', () => {
   expect.assertions(1);
-  return db.find().then((data) => {
+  return db.find({ 'place_id': '0' }).then((data) => {
     expect(Array.isArray(data)).toBe(true);
   });
 });
 
-test('the photos data is an array of length 10', () => {
+test('the photos data is an array with a length between 10 and 20', () => {
   expect.assertions(1);
```

Comment:
I know this isn't your code, but what does expect.assertions(1) do?

Collaborative answer between myself and Steve: 

-- Verify only one assertion in test. Useful when making async calls.